android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/wechat/WechatSocialHelper.kt

```diff
@@ -0,0 +1,12 @@
+package io.logto.sdk.android.auth.social.wechat
+
+import android.app.Activity
+import com.tencent.mm.opensdk.openapi.IWXAPI
+import com.tencent.mm.opensdk.openapi.WXAPIFactory
+
+internal object WechatSocialHelper {
+    fun getApi(context: Activity, appId: String): IWXAPI =
+        WXAPIFactory.createWXAPI(context, appId).apply {
+            registerApp(appId)
+        }
+}
```

Comment:
don't need this extra object? we can just put it into result activity.

Reply:
updated